uid-pages/form-case-overview/src/caseoverview.json

```diff
@@ -1406,11 +1406,24 @@
         "return  (!$data.openCase && $data.archivedCase.length > 0) ? $data.archivedCase[0] : $data.openCase;"
       ],
       "exposed" : false
+    },
+    "initiator" : {
```

Comment:
This Code should be Factorized with the code of "Task executed by System"

Reply:
:heavy_check_mark: